flink-connectors/flink-connector-jdbc/src/test/resources/log4j2-test.properties

```diff
@@ -21,6 +21,14 @@
 rootLogger.level = OFF
 rootLogger.appenderRef.test.ref = TestLogger
 
+# debugging FLINK-22889
+logger.xa.name = org.apache.flink.connector.jdbc.xa.JdbcExactlyOnceSinkE2eTest
+logger.xa.level = DEBUG
+logger.task.name = org.apache.flink.runtime.taskmanager.Task
+logger.task.level = WARN
+logger.jdbc.name = org.apache.flink.connector.jdbc
+logger.jdbc.level = ERROR
```

Comment:
This would be reduced to the default INFO on AZP. Is this intended?

Reply:
No, I'll remove it. Thanks for pointing out.